sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -0,0 +1,64 @@
+{{ header }}
+
+{% if init %}
+CREATE TABLE IF NOT EXISTS
+  `{{ events_stream_table }}`
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id,
+  submission_date
```

Comment:
Given that we already partition on `submission_date`, what value do we get from clustering on `submission_date`? Thinking about the query patterns most common, what do you think about clustering on event name and or client id?

Reply:
I guess clustering on event name could make sense. One drawback with it could be that we have a relatively large number of different events. The more distinct values there are for clustering, the larger the metadata that needs to be scanned when querying which could potentially slow things down as well. Usually we clustered on event category which was a good option.